### ISSUE TITLE
Room for webrtc?

### ISSUE THREAD

**Issue by lmangani** [pfrazee/ctzn#26]
Hey @pfrazee 

Representing the group at https://github.com/meething where our cup of tea is decentralization of webrtc/conferencing.

Would you be open to provide some guidance geared towards receiving back contributions for a `room` concept for p2p real-time a/v conferences between ctzns? All we would need is an hyperbeam/web style channel _(analogy to a community i guess, but for ephemeral/realtime exchange, no memory/storage)_ ctzn users could create/join to discover others, exchange minimal presence signaling and we can bring the rest of the work to create a p2p audio/video rooms. Think a little lightweight built in mesh "ctznhouse" as a starting point and growing from there.

Any existing plans we could join and/or appeal/potential/space to help us try hack this in? Thanks in advance!

**Comment by pfrazee** [comment]
Yes! I think a WebRTC signalling protocol in the CTZN servers would be great. Let's slate this for post-1.0.
